MoviesMVVM/MoviesMVVM/View/View/MoviesViewController.swift

```diff
@@ -80,6 +80,12 @@ final class FirstViewController: UIViewController {
         return tableView
     }()
 
+    private var moviesViewData: MoviesViewData = .initial {
+        didSet {
+            view.setNeedsLayout()
+        }
+    }
+
     // MARK: Life cycle
 
     override func viewDidLoad() {
```

Comment:
```suggestion
    override func viewDidLoad() {
```

Убери если не используешь его